src/core_plugins/timelion/server/series_functions/aggregate/index.js

```diff
@@ -0,0 +1,41 @@
+import alter from '../../lib/alter.js';
+import Chainable from '../../lib/classes/chainable';
+import _ from 'lodash';
+
+const functions = {
+  avg: require('./avg'),
+  cardinality: require('./cardinality'),
+  min: require('./min'),
+  max: require('./max'),
+  last: require('./last'),
+  first: require('./first'),
+  sum: require('./sum')
```

Comment:
They're separate because I reuse them in a function I haven't submitted a pull for yet: https://github.com/rashidkpc/timelion-extras/blob/master/functions/orderby/index.js